js/ui/tree_view/ui.tree_view.base.js

```diff
@@ -719,7 +719,7 @@ const TreeViewBase = HierarchicalCollectionWidget.inherit({
         }
     },
 
-    _setAriaSelected: function() {},
+    _setAriaSelectionAttribute: function() {},
```

Comment:
Only tree_view.base and ui.menu_base inherit from `HierarchicalCollectionWidget`, so `_setAriaSelectionAttribute: noop` can be moved to the `HierarchicalCollectionWidget` and can be removed from this files

Reply:
i did not refactor anything here, just renamed the function